.jenkins/common_job_properties.groovy

```diff
@@ -162,8 +162,11 @@ class common_job_properties {
   // Sets common config for Maven jobs.
```

Comment:
if you don't mind, I'd pull this into its own PR. The rest are almost no-ops, this one seems like a behavior change that might need more thought/review.